ocs_ci/framework/__init__.py

```diff
@@ -13,6 +13,7 @@
 import logging
 from collections.abc import Mapping
 from dataclasses import dataclass, field, fields
+from ocs_ci.ocs.constants import HCI_CLIENT, HCI_PROVIDER
```

Comment:
As use of `from ocs_ci.ocs.constants import HCI_CLIENT, HCI_PROVIDER` lib inside `init.py` file causes circular import Error.  with Reference to other code in init.py file , using `hci_client` and `provider` as a hardcoded value.

```ocs_ci/ocs/constants.py:1045: in <module>
[58](https://github.com/red-hat-storage/ocs-ci/actions/runs/6866895930/job/18674124645?pr=8824#step:6:61)
Tue, 14 Nov 2023 19:04:00 GMT
from ocs_ci.utility.utils import get_ocp_version
[59](https://github.com/red-hat-storage/ocs-ci/actions/runs/6866895930/job/18674124645?pr=8824#step:6:62)
Tue, 14 Nov 2023 19:04:00 GMT
ocs_ci/utility/utils.py:41: in <module>
[60](https://github.com/red-hat-storage/ocs-ci/actions/runs/6866895930/job/18674124645?pr=8824#step:6:63)
Tue, 14 Nov 2023 19:04:00 GMT
from ocs_ci.framework import config
[61](https://github.com/red-hat-storage/ocs-ci/actions/runs/6866895930/job/18674124645?pr=8824#step:6:64)
Tue, 14 Nov 2023 19:04:00 GMT
E ImportError: cannot import name 'config' from partially initialized module 'ocs_ci.framework' (most likely due to a circular import) (/home/runner/work/ocs-ci/ocs-ci/ocs_ci/framework/__init__.py)```